src/syscheckd/syscheck_audit.c

```diff
@@ -479,6 +498,12 @@ int audit_init(void) {
     audit_added_dirs = W_Vector_init(20);
 
     add_audit_rules_syscheck(true);
+
+    // Change to realtime directories that don't have any rules when Auditd is in immutable mode
+    if (audit_status == AUDIT_IMMUTABLE) {
```

Comment:
Related to the previous comment, `audit_status` is of type int but we are checking its value to `AUDIT_IMMUTABLE`, can we change its type?